src/main/java/de/tum/in/www1/artemis/service/util/RoundingUtil.java

```diff
@@ -19,6 +19,18 @@ public static double round(double number) {
         return roundToNDecimalPlaces(number, 1);
```

Comment:
This method can also be deleted. Only the `float` variant seems to be doing some type magic.

Reply:
I think there was a reason, why `roundToNDecimalPlaces` is not public. I guess we want to always round to 1 decimal place? 
Having this at a central place avoids to always call the function with N = 1. Some of the latest changes have added the `round()` function. @stefanwaldhauser am I correct and what do you think?